cypress/integration/a11y/global_variables.js

```diff
@@ -0,0 +1,20 @@
+const viewports = ['iphone-5', 'macbook-13']
+
+const A11Y_CONFIG = {
+  reporter: "v2"
+}
+
```

Comment:
Why do we need to exclude constants? Are all constants need to be exlucded?

Reply:
Not sure I understand the question? This is the config option that gets pushed into `axe` during testing, via `cy.injectAxe().configureAxe(A11Y_CONFIG)`

Reply:
I think these are excludes for JSX components? I'm not actually sure... we probably want to remove those as we active start running a11y tests